<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>402</width>
    <height>516</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>402</width>
    <height>516</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../logo/Wikimedia_Brand_Guidelines_Update_2022_Wikimedia_Logo_Brandmark.png</normaloff>../logo/Wikimedia_Brand_Guidelines_Update_2022_Wikimedia_Logo_Brandmark.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Iniciar Sesión</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UsuarioEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(74, 30, 134, 0);
border: none;
border-bottom: 3px solid rgba(85, 0, 255, 200);
border-radius: 5
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Nombre Usuario</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ContrasenaEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>280</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(74, 30, 134, 0);
border: none;
border-bottom: 3px solid rgba(85, 0, 255, 200);
border-radius: 5
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
     <property name="placeholderText">
      <string>Contraseña</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BotonAceptar">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(182, 119, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-radius: 10
</string>
     </property>
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>241</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(220, 220, 220, 165);
border-radius: 10
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>401</width>
       <height>601</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/prefijoNuevo/foto inicial.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>-80</x>
       <y>0</y>
       <width>481</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../recursos.qrc">:/imagenes/foto_inicial.jpg</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="BotonRecuContra">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(182, 119, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-radius: 10
</string>
     </property>
     <property name="text">
      <string>Recuperar contraseña</string>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label</zorder>
    <zorder>UsuarioEdit</zorder>
    <zorder>ContrasenaEdit</zorder>
    <zorder>BotonAceptar</zorder>
    <zorder>BotonRecuContra</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
